gameengine.js

```diff
@@ -7,7 +7,7 @@ window.requestAnimFrame = (function requestAnimFrame() {
             || window.mozRequestAnimationFrame
             || window.oRequestAnimationFrame
             || window.msRequestAnimationFrame
-            || function animFrame(/* function */ callback, /* DOMElement */ element) {
```

Comment:
For this line ESlint mentioned that element was defined and never used and considering that seems to be true I went ahead and deleted it